central/compliance/data/repository.go

```diff
@@ -256,10 +261,21 @@ func (r *repository) init(ctx context.Context, domain framework.ComplianceDomain
 		r.scanners = append(r.scanners, scannerIntegration.GetScanner())
 	}
 
-	r.processIndicators, err = f.processIndicatorStore.SearchRawProcessIndicators(ctx, clusterQuery)
+	sshProcessQuery := search.NewQueryBuilder().AddRegexes(search.ProcessExecPath, ".*ssh.*").ProtoQuery()
```

Comment:
Should go ahead and check for ssh at the end of the path? `^.*\/?ssh$`

Reply:
ah yeah, we need to check for sshd also.

Reply:
There's also the alternative ssh servers: [dropbear](https://matt.ucc.asn.au/dropbear/dropbear.html) and [lsh](https://www.lysator.liu.se/~nisse/lsh/).

Reply:
These are all good points and I'm not sure of the ideal solution generally; however, perhaps we can decouple it from this PR as this basically emulates exactly what the code does today with performance gains